service/awsconfig/v4/key/key.go

```diff
@@ -41,6 +45,45 @@ func BucketObjectName(templateVersion string, prefix string) string {
 	return fmt.Sprintf("cloudconfig/%s/%s", templateVersion, prefix)
 }
 
+func CloudConfigSmallTemplates() []string {
+	return []string{
+		cloudconfig.Small,
+	}
+}
```

Comment:
The new functions here are basically getters for the templates we need to render. The rendering became a more general thing with a more common interface. That way it is simpler to render templates. You only have to provide the list of templates (content) and data you want to interpolate eventually. No other magic or difference between different template renderings. Note that we cannot have constants for lists and I felt safer having an immutable getter instead of variables that can be screwed with.

Reply:
Nice, I like the structure here.